config/screen_config.py

```diff
@@ -0,0 +1,15 @@
+# Dashboard configurations
+# FONT_SIZE =
+# FONT_FAMILY =
+# FONT_COLOR =
+# DASHBOARD_BG_COLOR = 
+
+# Pop-up message configurations
+# FONT_SIZE_POPUP =
+# FONT_COLOR_POPUP =
+
+# GOOD_TAP_BG_COLOR =
+# BAD_TAP_BG_COLOR =
+
+# SET_ORDER_BG_COLOR =
```

Comment:
- Prepended all general pop-up configurations with `POPUP_` for consistency. 
- Added `POPUP_ERROR_BG_COLOR` and `POPUP_WARNING_BG_COLOR` to represent errors (e.g. NFC read failed) and warnings (e.g. tap within debounce time)
- Since there is already representation for error and warning, only good NFC and touch sensor taps configurations were created. Here:
```py
# Tap event pop-up configurations
TAP_EVENT_BG_COLOR = ''

# NFC tap pop-up configurations
ORDER_SET_BG_COLOR = ''
EMPLOYEE_SET_BG_COLOR = ''
```

Reply:
Alternative:
GOOD_ORDER_NFC_BG_COLOR
GOOD_EMPLOYEE_NFC_BG_COLOR